x/auth/keeper/grpc_query.go

```diff
@@ -24,8 +24,12 @@ func (ak AccountKeeper) AccountAddressByID(c context.Context, req *types.QueryAc
 		return nil, status.Errorf(codes.InvalidArgument, "empty request")
 	}
 
+	if req.Id < 0 {
+		return nil, status.Error(codes.InvalidArgument, "Invalid account id")
```

Comment:
If the better name is account number, shouldn't we reflect that in the error message?